src/org/sosy_lab/java_smt/example/NQueens.java

```diff
@@ -106,14 +105,15 @@ BooleanFormula[][] getSymbols() {
   // the N-Queens problem
   List<BooleanFormula> getRules(BooleanFormula[][] symbols) {
     List<BooleanFormula> rules = new ArrayList<>();
-    int symbolLength = symbols.length;
     /* Rule 1: At least one queen per row,
          or we can say make sure that there are N Queens on the board
     */
     for (BooleanFormula[] rowSymbols : symbols) {
-      List<BooleanFormula> clause =
-          new ArrayList<>(Arrays.asList(rowSymbols).subList(0, symbolLength));
-      rules.add(this.bmgr.or(clause));
+      List<BooleanFormula> clause = new ArrayList<>();
+      for (int i = 0; i < n; i++) {
```

Comment:
well,... that is also valid now.

Reply:
Maybe I should switch to Linux, but then again......